testData/solutions/win32.json5

```diff
@@ -733,13 +959,16 @@
                                 "off",
                                 "on"
                             ],
-                            "default": 1
+                            "default": 0
                         }
                     },
-                    "HTML.SmartNavigation": {
+
+                    // ----------- WEB / HTML / PDF - Tables -------------
+
+                    "HTML.IndicateColSpan": {
                         "schema": {
-                            "title": "Smart Navigation",
-                            "description": "Whether to enable \"smart navigation\" when reading web pages.",
+                            "title": "Indicate ColSpan for Braille",
```

Comment:
"for Braille" and "when reading" seem contradictory to me.  Is this only for braille readers, or for announced content?

Reply:
I think this must have been a type, changed the title to just Indicate ColSpan.